src/Overlay/Modal/Modal.styled.ts

```diff
@@ -0,0 +1,82 @@
+import styled, { css, keyframes } from 'styled-components'
+import { Button } from '../../Button'
+import { titleLarge } from '../../theme'
+
+
+const fadeInAnimation = keyframes`
+  0% {
+    opacity: 0;
+    transform: scale(0);
```

Comment:
The animation looks good, just need some tweaks to make it less jarring.

scale: 0.8
opacity: 0.3

Reply:
I have updated the values as you suggested.